accordion.js

```diff
@@ -162,13 +158,23 @@
 		allTitles.keydown(function(e) {
 			var code = e.which;
 
-			// 13 = Return, 32 = Space
+			// 13 = Return, 32 = Space, 27 = Esc
 			if ((code === 13) || (code === 32)) {
 				// Simulate click on title
 				$(this).click();
 			}
 		});
 
+		allTitles.on('keyup', function(e) {
```

Comment:
Is there a reason to use `keyup` instead of `keydown` (like the others) for esc key?

Reply:
It seems like using keyup, the accordion content won't slide up until the Esc key is released, versus as soon as the key is hit. That to me seems like closing the content is more intentional, vs. accidentally pressing the Esc when brushing over it and then the content closes. It's not really that logical of an answer though, more UX than anything. 

If it's more consistent to use keydown and the animation doesn't change much, then I can certainly use keydown instead of keyup.